.gitignore

```diff
@@ -107,3 +107,5 @@ _UpgradeReport_Files/
 Backup*/
 UpgradeLog*.XML
 *.nupkg
+.vscode/launch.json
```

Comment:
IDE config files should be added to the **global** `.gitignore` instead of the repository `.gitignore`.

Reply:
Yes, you may be right. 

However, in this `.gitingore` we already had Visual Studio's users files ignored. So I thought it could be a good idea to add files generated from VSCode to prevent contributors to push their user files.

For example a famous .NET library JSON.Net has those folders ignored in their [gitignore](https://github.com/JamesNK/Newtonsoft.Json/blob/master/.gitignore)

Reply:
Thanks @Ant-hem for mention those famous examples. My comment is just a php thing. 😅

Reply:
@nunomaduro I think it's also a really good idea to have a global `.gitignore` file 😄 
The thing is, if one you contributor doesn't, he could push undesired files.